### PR TITLE
Support D'CENT wallet in PC and mobile

### PR DESCRIPTION
**Pull request by KyungeunKim**
### Description
Support D'CENT wallet in PC and mobile 

### Checklist
- [ ] The version field in `package.json` is incremented following [semantic versioning](https://semver.org/)
- [ ] The box that allows repo maintainers to update this PR is checked
- [ ] I tested locally to make sure this feature/fix works
- [ ] This PR passes the Circle CI checks

### CODE REVIEW

**Review by ramsemune123 — COMMENTED** [review]
#992

**Comment by ramsemune123** [comment]
#992

File: packages/dcent/package.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
+{
+  "name": "@web3-onboard/dcent",
+  "version": "1.0.0",
```
Comment (Adamj1232) [inline packages/dcent/package.json:3]:
```suggestion
  "version": "1.0.0-alpha.1",
```

**Comment by Adamj1232** [comment]
@KyungeunKim thank you for the PR! I updated the base branch to our new develop branch (`v2-web3-onboard-develop`) to allow for testing in the new staging env. With this if you could please merge in that develop branch to handle any conflicts and then update the version in the package.json as shown above

File: README.md [hunk 2]
```diff
@@ -77,7 +77,7 @@ For full documentation, check out the README.md for each package:
 - [Ledger](packages/ledger/README.md)
 - [Trezor](packages/trezor/README.md)
 - [Keystone](packages/keystone/README.md)
-
+- [D'CENT](packages/dcent/README.md)
```
Comment (Adamj1232) [inline README.md:80]: Will need to add a README for D'Cent describing usage and init options, similar to the format of other package's READMEs

Comment (KyungeunKim) [reply README.md:80]: @Adamj1232 
I added "README". Please review again.

File: packages/dcent/yarn.lock [hunk 3]
```diff
@@ -0,0 +1,1154 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
Comment (Adamj1232) [inline packages/dcent/yarn.lock:1]: We can remove this file as we use the yarn.lock at the root of the project

File: README.md (hunk 2 above)
Comment (Adamj1232) [reply README.md:80]: Perfect, thank you!!

**Review by Adamj1232 — CHANGES_REQUESTED** [review]

File: packages/dcent/src/index.ts [hunk 4]
```diff
@@ -0,0 +1,303 @@
...
+
+        const { StaticJsonRpcProvider } = await import(
+          '@ethersproject/providers'
+        )
+        const { default: Common, Hardfork } = await import('@ethereumjs/common')
+
+        const { default: EthDcentKeyring } = await import('eth-dcent-keyring')
+        const dcentKeyring = new EthDcentKeyring({})
+        
+        console.log('dcentKeyring ', dcentKeyring)
```
Comment (Adamj1232) [inline packages/dcent/src/index.ts:93]: Can we remove this console log?

**Comment by KyungeunKim** [comment]
@Adamj1232 
Thank you for your review.
I removed yarn.lock and console.log and merge v2-web3-onboard-develop branch again.

File: packages/dcent/src/index.ts [hunk 5]
```diff
@@ -0,0 +1,303 @@
...
+    address: addressList[0],
+    balance: {
+      asset: '',
+      value: await provider.getBalance(addressList[0])
+    }
+  }
+
+  accounts.push(account)
+  
+  // console.log('generateAccounts/accounts = ', accounts)
```
Comment (Adamj1232) [inline packages/dcent/src/index.ts:52]: Can we remove this commented out code?

Comment (Adamj1232) [inline packages/dcent/src/index.ts:52]: Can we remove this commented out code?

Comment (Adamj1232) [inline packages/dcent/src/index.ts:52]:
```suggestion
```

**Review by Adamj1232 — APPROVED** [review]

**Pull request merged**